.github/workflows/all_solutions.yml

```diff
@@ -26,19 +33,24 @@ env:
   DOTNET_NOLOGO: true
 
 jobs:
-  build-profiler:
-    uses: ./.github/workflows/build_profiler.yml
-    secrets: inherit
-    permissions:
-      pull-requests: read
-      contents: write
-      packages: write
+
+  # build-profiler:
```

Comment:
Keeping this in for any reason?

Reply:
See my comment on @nrcventura's comment... ;-)